package.json

```diff
@@ -93,6 +93,7 @@
     "react-beautiful-dnd": "^4.0.1",
     "react-dom": "^16.9.0",
     "react-focus-lock": "^2.2.1",
+    "react-resizable": "^1.10.1",
```

Comment:
I have a hatred for this library from my poc work on it :'(

Reply:
it's great if you're doing something weird like using the grid. Awful in every other way.